recipes/scikit-multiflow/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "scikit-multiflow" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "69fa62479d6b4cf3aa04800fde3c1cf012c3350fff1cc6f5648a0e646ecbfc5d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script:
+    - "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - pytest
+    - pytest-runner
+    - scipy >1.0.0
+    - scikit-learn >=0.20
+    - sortedcontainers >=1.5.7
+    - matplotlib >=2.0.0
+    - pandas >=0.21.0
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >1.0.0
+    - scikit-learn >=0.20
+    - sortedcontainers >=1.5.7
+    - matplotlib >=2.0.0
+    - pandas >=0.21.0
+
+test:
+  imports:
+    - skmultiflow
+  commands:
+    - conda inspect linkages -p $PREFIX {{ name }}  # [not win]
```

Comment:
This isn't needed any more as `conda-build` does this check automatically.

Reply:
Removed in f31ce0c380863ab695530ba37e0b80ed3aa3ec5d